File: CP008/Java/Main.java
package fosalgo;

import java.util.Scanner;
class Main {
    public static void main(String[] args) {
        int a,b;
        Scanner sc = new Scanner(System.in);
        a = sc.nextInt();
        b = sc.nextInt();
        if(a<b){
            System.out.println("MIN = "+a);
            System.out.println("MAX = "+b);
        }else{
            System.out.println("MIN = "+b);
            System.out.println("MAX = "+a);
        }
    }
}

File: CP008/C++/Main.cpp
#include <iostream>
using namespace std;
int main() {
    int a,b;
    cin>>a;
    cin>>b;
    if(a<b){
        cout<<"MIN = "<< a <<"\n";
        cout<<"MAX = "<< b <<"\n";
    }else{
        cout<<"MIN = "<< b <<"\n";
        cout<<"MAX = "<< a <<"\n";
    }
    return 0;
}
